code/modules/paperwork/pen.dm

```diff
@@ -137,7 +138,7 @@ var/paperwork_library
 	return "<span style=\"[style];color:[P.color]\">[t]</span>"
 
 /datum/writing_style/pen/New()
-	addReplacement(REG_BBTAG("*"), "<li>")
```

Comment:
You can't make lists using the asterisk, if anyone wants to poke around and fix it go ahead I had to manually fix most of these regex expressions because none of them worked already.